packages/dd-trace/src/appsec/sdk/utils.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+function getRootSpan (tracer) {
+  const span = tracer.scope().active()
+  if (span) return span.context()._trace.started[0]
```

Comment:
actually i'm stupid:
```suggestion
  return span && span.context()._trace.started[0]
```